app/views/components/config/DissociateUcrConfig.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.components.config
+
+import config.DucrPartConfig
+import javax.inject.Inject
+import play.api.mvc.Call
+
+class DissociateUcrConfig @Inject()(ducrPartConfig: DucrPartConfig) {
+
+  def backUrl: Call =
+    if (ducrPartConfig.isDucrPartsEnabled)
```

Comment:
I'm not sure if this back link is correct, similar thoughts like above, that there is missing case, but not sure for what page this config is

Reply:
So probably the same answer like above, it's okay :)